android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -285,6 +288,20 @@ public void onHostDestroy() {
       cleanupCallbacksAndDestroy();
     }
 
+    @Override
+    protected void onSizeChanged(int w, int h, int ow, int oh) {
+      if (sendContentSizeChangeEvents) {
+        dispatchEvent(
+          this,
+          new ContentSizeChangeEvent(
+            this.getId(),
+            this.getWidth(),
```

Comment:
Can't use any of the variables above ?

Reply:
Potentially I guess, I just wasn't sure which ones they relate to as the former implementation relied on `getContentHeight()` and not sure if this is the same value as provided by `int height`

Thoughts? Can swap it over if needed?

Reply:
Well I'd rather use the variables provided instead of mesuring again, if possible